src/Stryker.Core/Stryker.Core/Mutants/NodeOrchestrators/NodeSpecificOrchestrator.cs

```diff
@@ -89,35 +94,94 @@ protected NodeSpecificOrchestrator(CsharpMutantOrchestrator mutantOrchestrator)
         /// <remarks>Override this method if you want to control how the node's children are mutated. simply return <see cref="node"/> if you want to
         /// skip mutation the children node.</remarks>
         protected virtual TBase OrchestrateChildrenMutation(TNode node, MutationContext context) =>
-            node.ReplaceNodes(node.ChildNodes(), 
-                (original, _) => MutantOrchestrator.Mutate(original, context));
+            node.ReplaceNodes(node.ChildNodes(),
+                computeReplacementNode: (original, _) => MutateSingleNode(original, context));
+
+        protected virtual SyntaxNode MutateSingleNode(SyntaxNode node, MutationContext context) => context.FindHandler(node).Mutate(node, context);
+
+        protected virtual MutationContext PrepareContext(TNode node, MutationContext context)
+        {
+            foreach (var commentTrivia in node.GetLeadingTrivia().Where(t => t.IsKind(SyntaxKind.SingleLineCommentTrivia) || t.IsKind(SyntaxKind.MultiLineCommentTrivia)).Select(t => t.ToString()))
+            {
+                // perform a quick pattern check to see if it is a 'Stryker comment'
+                if (!Pattern.Match(commentTrivia).Success)
+                {
+                    continue;
+                }
+                var match = Parser.Match(commentTrivia);
```

Comment:
I'm not very familiar with Regex performance. Does it actually save anything to first do a quick match before the full match?

Reply:
This is not related to performance. This first regex is used to recognize special comments so we can raise an error to the user if there is an error in the comment syntax. 
Check could be made the other way around : check the complex first, then the simple one, but it would mean testing twice each regular comment.